File: src/main/java/net/botlify/anticaptchacom/response/QueueStatsResponse.java
package net.botlify.anticaptchacom.response;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;
import org.jetbrains.annotations.NotNull;

@Getter
@ToString
@EqualsAndHashCode
@NoArgsConstructor(force = true, access = AccessLevel.PRIVATE)
public class QueueStatsResponse {

  /**
   * Number of idle workers online, waiting for a task.
   */
  @JsonProperty("waiting")
  public int waiting;

  /**
   * Queue load as a percentage.
   */
  @JsonProperty("load")
  public double load;

  /**
   * Average task solution cost in USD.
   */
  @JsonProperty("bid")
  public double bid;

  /**
   * Average task solution speed in seconds.
   */
  @JsonProperty("speed")
  public double speed;

  /**
   * Total number of workers.
   */
  @JsonProperty("total")
  public int total;

}

File: src/main/kotlin/net/botlify/anticaptchacom/data/response/QueueStatsResponse.kt
package net.botlify.anticaptchacom.data.response

import com.fasterxml.jackson.annotation.JsonProperty

data class QueueStatsResponse(
    /**
     * Number of idle workers online, waiting for a task.
     */
    @JsonProperty("waiting")
    var waiting: Int,

    /**
     * Queue load as a percentage.
     */
    @JsonProperty("load")
    var load: Double,

    /**
     * Average task solution cost in USD.
     */
    @JsonProperty("bid")
    var bid: Double,

    /**
     * Average task solution speed in seconds.
     */
    @JsonProperty("speed")
    var speed: Double,

    /**
     * Total number of workers.
     */
    @JsonProperty("total")
    var total: Int
);